main.go

```diff
@@ -150,6 +150,10 @@ func main() {
 			Name:  "fsuid-map-fail-on-error",
 			Usage: "When set to true, fail to launch a container whenever filesystem uid-mapping (needed for files to show proper ownership inside the container's user-namespace) hits an error; when set to false, launch the container anyway (files may show up owned by nobody:nogroup) (default = false).",
 		},
+		cli.BoolTFlag{
```

Comment:
A couple of questions:

* Why not allow this behavior by default through this knob?
* Why not remove this knob and allow this behavior for all sysbox containers for kernels +6.7?